### PR TITLE
[ENH] Add computational models and mega-analysis BEPs

### PR DESCRIPTION
**Pull request by tsalo**
This PR adds rows for [BEP034 (computational models)](https://docs.google.com/document/d/1NT1ERdL41oz3NibIFRyVQ2iR8xH-dKY-lRCB4eyVeRo/edit#) and [BEP035 (mega-analysis)](https://docs.google.com/document/d/10RpFFsG_ESj0orGIqJ0yK4DTpeoyrmd4RjX2ER-P31k/edit#) to the BEP table in `get_involved.md`.

I don't know the appropriate "summary" for each (i.e., where they are in the process and what they need from contributors).

### CODE REVIEW

**Comment by effigies** [comment]
I've linked this PR on the doc. Let's see if they want to adjust the contacts or provide a quick blurb.

**Review by sappelhoff — APPROVED** [review]

**Comment by spisakt** [comment]
BEP035 (mega-analysis): contacts and summary OK form our side!

**Comment by tsalo** [comment]
The CI failures seem to be due to the fact that the new pages aren't on the website, so that should be fine _after_ it's merged. We just need descriptions of contribution opportunities for the new BEPs before this will be ready to merge.

File: _pages/get_involved.md
Comment (spisakt) [inline _pages/get_involved.md:85]: New BEP, collecting comments and community feedback, especially on handling "conditions-of-interests". Collaborators with mega-analysis experience or with actual example use-cases (planned/ongoing/finished mega analyses) welcome!

Comment (tsalo) [reply _pages/get_involved.md:85]: Thanks!

**Review by effigies — APPROVED** [review]

**Comment by effigies** [comment]
Should also ping @MichaelSchirner for a status blurb.

**Comment by MichaelSchirner** [comment]
Thanks for adding the computational models BEP034! Contacts OK from our side. As summary we suggest: "BEP Computational Models: PR open for review and discussion [here](https://github.com/bids-standard/bids-specification/pull/850)"

**Comment by tsalo** [comment]
Now that we have all of the components (and two approvals), is this good to merge?

**Comment by franklin-feingold** [comment]
lgtm too - thank you @tsalo !

**Pull request merged**
